modules/components/stories/Aggs.js

```diff
@@ -187,7 +182,10 @@ storiesOf('Aggs', module)
       <DatesAgg
         field="disease_type"
         displayName="Disease Type"
-        buckets={require('./dummyData/datesBucketsSample.json')}
+        stats={{
+          min: 1529539200125,
```

Comment:
could you put this number and the bottom one as named consts maybe `const JUN_19`? Just a bit magic numbery

Reply:
I think I'll leave a comment cause I kinda wanna use the story as documentation as to what to pass in here 📦